src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -31,6 +60,162 @@
         az sf application certificate add -g group-name -n cluster1  --secret-identifier 'https://{KeyVault}.vault.azure.net/secrets/{Secret}'
 """
 
+helps['sf application show'] = """
+type: command
+short-summary: Get an application.
+examples:
+  - name: Get application.
+    text: >
+        az sf application show -g testRG -n testCluster --application-name testApp
+"""
+
+helps['sf application list'] = """
+type: command
+short-summary: List applications of a given cluster.
+examples:
+  - name: List applications for a given application type.
+    text: >
+        az sf application list -g testRG -n testCluster
+"""
+
+helps['sf application delete'] = """
+type: command
+short-summary: Delete an application type.
+examples:
+  - name: Delete application type.
+    text: >
+        az sf application delete -g testRG -n testCluster --application-name testApp
+"""
+
+helps['sf application-type'] = """
+type: group
+short-summary: Manage application types on an Azure Service Fabric cluster.
+"""
+
+helps['sf application-type create'] = """
+type: command
+short-summary: Create a new application type on an Azure Service Fabric cluster.
+examples:
+  - name: Create new application type.
+    text: >
+        az sf application-type create -g testRG -n testCluster --application-type-name testAppType
+"""
+
+helps['sf application-type show'] = """
+type: command
+short-summary: Get an application type.
+examples:
+  - name: Get application type.
+    text: >
+        az sf application-type show -g testRG -n testCluster --application-type-name CalcServiceApp
+"""
+
+helps['sf application-type list'] = """
+type: command
+short-summary: List application types of a given cluster.
+examples:
+  - name: List application types for a given cluster.
+    text: >
+        az sf application-type list -g testRG -n testCluster
+"""
+
+helps['sf application-type delete'] = """
+type: command
+short-summary: Delete an application type.
+examples:
+  - name: Delete application type.
+    text: >
+        az sf application-type delete -g testRG -n testCluster --application-type-name CalcServiceApp
+"""
+
+helps['sf application-type-version'] = """
```

Comment:
What about using subgroup `az sf application-type version` ?

Reply:
Agree. Please check our command guide line. https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior

Reply:
changed version to ve subgroup of application-type